lib/ws_servers/api/algos/algo_worker.js

```diff
@@ -2,11 +2,12 @@
 
 const debug = require('debug')
 const d = debug('bfx:hf:server:algo-worker')
-
 const { AOHost } = require('bfx-hf-algo')
+const { RESTv2 } = require('bfx-api-node-rest')
 
 const validateAO = require('./util/validate_ao')
 const { DMS_ENABLED } = require('../../../constants')
+const { auth: { tokenTtlInSeconds } } = require('../../../../config/algo_server.conf.json')
```

Comment:
maybe inject into constructor via settings to keep it easier testable as without requiring the config.json?

Reply:
done